frontend/src/components/Room.tsx

```diff
@@ -1,179 +1,316 @@
+        <div className="flex flex-col items-center text-center">
+            <h1 className="text-3xl font-bold mb-4">Sala</h1>
+            {isRoomFetching ? (<p className="text-center">Ładowanie danych...</p>) : (<div className="w-60 text-start">
+                <p className="pb-4 text-4xl text-center font-bold">
+                    {room?.number}
+                </p>
+
+                {!token ? (<p className="mb-2 text-lg">
+                    <span className="font-bold">Rodzaj: </span>
+                    {room?.roomAnnotation}
+                </p>) : (<div className="mb-2">
+                    <label className=" text-lg font-bold">
+                        Rodzaj:
+                        <select
+                            value={Object.entries(RoomAnnotation).find(([key, val]) => val === roomAnnotation)?.[0]}
+                            onChange={roomAnnotationChange}
+                            className="ml-1  font-normal text-center px-1 border-b-2 border-blue-500
+                                    hover:border-2 hover:border-blue-500 hover:rounded-md  focus:border-2
+                                    focus:border-blue-500 focus:rounded-md outline-none "
+                        >
+                            {Object.keys(RoomAnnotation).map((key) => (
+                                <option value={key}>{RoomAnnotation[key]}</option>))}
+                        </select>
+                    </label>
+                </div>)}
+
+                <p className="mb-2 text-lg">
+                    <span className="font-bold">Pojemność: </span>
+                    {room?.capacity}
+                </p>
+            </div>)}
+        </div>
+
+        <div>
+            {isRoomActivitiesFetching ? (<div className="mt-8 text-center">Ładowanie danych...</div>) : (<div>
+                <div className="flex flex-col my-8">
+                    <hr className="h-px bg-gray-200 border-0 h-0.5 dark:bg-gray-700"/>
+                    <div>
+                        <CalendarToPrint ref={calendarRef}/>
+                        <ReactToPrint
+                            trigger={() => <div className="center-with-margin">
+                                <button
+                                    className="hover:shadow-form rounded-md bg-sky-500 py-3 px-8 text-center text-base font-semibold text-white outline-none">
+                                    Wydrukuj plan
+                                </button>
+                            </div>}
+                            content={() => calendarRef.current}
+                            onAfterPrint={() => {
+                                setCbxSelectDateChecked(false);
+                                window.scrollTo(0, 0);
+                            }}
+                        />
+                        <div className="center-with-margin">
+                            <Checkbox
+                                label="Usuń nagłówki dat"
```

Comment:
should only delete day of month number, for example "26 pon" should be changed to "pon" and display only this

Reply:
Done